packages/hackathon/src/scenes/HomeScene/HomeScene.tsx

```diff
@@ -61,7 +63,7 @@ export const HomeScene: FC<HomeSceneProps> = ({ hacker }) => {
             Agenda
           </Heading>
           <Paragraph>
-            <ExtMarkdown source={intro} />
+            <ExtMarkdown source={intro} linkClickHandler={onClick} />
```

Comment:
It's strange that you needed this, because ExtMarkdown handles link clicks already for extensions.